packages/ui/components/switch/src/LionSwitchButton.js

```diff
@@ -158,8 +162,18 @@ export class LionSwitchButton extends DisabledWithTabIndexMixin(LitElement) {
    */
   requestUpdate(name, oldValue, options) {
     super.requestUpdate(name, oldValue, options);
-    if (this.isConnected && name === 'checked' && this.checked !== oldValue && !this.disabled) {
+    if (
+      this.__initialized &&
+      this.isConnected &&
+      name === 'checked' &&
+      this.checked !== oldValue &&
+      !this.disabled
+    ) {
       this.__checkedStateChange();
     }
   }
+
+  firstUpdated() {
```

Comment:
```suggestion
  /** @param {import('lit').PropertyValues } changedProperties */
  firstUpdated(changedProperties) {
    super.firstUpdated(changedProperties);
```

Reply:
Even though ]lit says it's not necessary to call `super`](https://lit.dev/docs/components/lifecycle/#firstupdated ), inside lion (complex inheritance hierarchy of mixins + superclasses) it is... 

It would be nice to have a linter for this